backend/core/src/shared/email/email.service.ts

```diff
@@ -148,6 +148,60 @@ export class EmailService {
     )
   }
 
+  public async newlisting(listing: Listing, user: User) {
+    await this.loadTranslationsForUser(user)
```

Comment:
Do you need to update the translations db for this? Or will that be in a separate PR?

Reply:
Yeah I was going to do this in a separate pr.